lib/functions-custom.php

```diff
@@ -49,4 +49,34 @@ function menuActiveClasses($item = null, $type = null, $classes = null) {
 
 function render_embed_caption($caption) {
   echo '<h5 class="text-align-center margin-top-tiny margin-bottom-tiny">' . $caption . '</h5>';
+}
+
+
+function render_quote($text = null, $person = null, $luminary = null) {
```

Comment:
i dont know if there will be another context not using theme options. But how will you tell the partial the context? There are 3 different groups theme options here for each page. One is a post type archive template the other to custom pages